src/views/v2/components/messages.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * View: Messages
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/components/messaging.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var array $messges An array of user-facing messages, managed by the View.
+ */
+
+if ( empty( $messages ) ) {
+	return;
+}
+
+// @todo @juanfra Make this beautiful.
+?>
+
+<div class="tribe-events-notices" style="text-align: center; width: 100%; margin: 0 0 1em auto;">
+	<?php foreach ( $messages as $message_type => $message_group ) {
+		foreach ( $message_group as $message ) {
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 6bd720b575535e646023cb70179cf0d3</sub>